zetasql/parser/ast_node_kind.h

```diff
@@ -150,6 +150,7 @@ enum ASTNodeKind {
   AST_GRANTEE_LIST,
   AST_GRANT_STATEMENT,
   AST_GRANT_TO_CLAUSE,
+  AST_GRANT_PATH,
```

Comment:
do not insert, append the new enum

Reply:
added by alphabetical order. are there compatibility issues?

Reply:
yes, enum are int, insert change the value of other enum entities.